submit.go

```diff
@@ -120,11 +120,13 @@ type submitResponse struct {
 	ReportURL string `json:"report_url,omitempty"`
 }
 
-// regex to catch and substitute ambiguous issue references with explicit ones to the actual repo they are in
-var ambiguousIssueRegex = regexp.MustCompile(`(^|[([{\s])(#\d+)([^\w]|$)`)
+// regex to match and substitute ambiguous issue references in the rageshake body text
+// matches a hash followed by digits optionally surrounded by whitespace or some punctuation
```

Comment:
this isn't really sufficient, I'm afraid. I'm looking for an explanation of how it works without having to stare hard at it and find the go regex docs.

```suggestion
//
// This matches:
//    (^|[([{\s]) - the start of the text, one of '(' or '[', '{', or a whitespace character
//    (?:^|#) - the start of the text, or a #
//    (\d+) - one or more digits
//    ([^\w]|$) - a non-word character (?) or the end of the text.
```